Shopping List/Shopping List/Controller/Table with shopping items/ItemsTableViewController.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  ItemsTableViewController.swift
+//  Shopping List
+//
+//  Created by Андрей Погосский on 3/2/20.
+//  Copyright © 2020 Андрей Погосский. All rights reserved.
+//
+
+import UIKit
+import os.log
+
+class ItemsTableViewController: UITableViewController {
+
+    // MARK: - Properties
+    let shoppingModel = ShoppingModel()
+
+    // MARK: - viewDidLoad()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        navigationItem.leftBarButtonItem = editButtonItem
+    }
+
+    // MARK: - Table view data source
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return shoppingModel.shoppingList.count
+    }
+
+    override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        return true
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+
+        let cellIdentifier = "ItemTableViewCell"
+
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier,
+                                                       for: indexPath) as? ItemTableViewCell  else {
+            fatalError("The dequeued cell is not an instance of ItemTableViewCell.")
```

Comment:
What happens if your app runs this code?

Reply:
Because the `dequeueReusableCell` return `UITableViewCell`, we need to downcast cell to the type of `ItemTableViewCell` to access to the property of `ItemTableViewCell` class. And if downcast fails, close the app with a specific error.

Reply:
Replaced `fatalerror` behavior with more soft solution, without app termination.